src/test/java/com/example/libraries/DepartmentServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package com.example.libraries;
+
+import EmployeeClass.Employee;
+import EmployeeServices.EmployeeService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class DepartmentServiceTest {
+    private Map<String, Employee> defaultMap;
+    private Employee firstEmployee;
+    private Employee secondEmployee;
+    private Employee thirdEmployee;
+    private Employee fourthEmployee;
+    @Mock
+    private EmployeeService employeeService;
+    @InjectMocks
+    private DepartmentServiceImpl out;
+    @BeforeEach
+    public void setUp(){
+        firstEmployee = new Employee("Ivan","Ivanov",1,100);
+        secondEmployee = new Employee("Marina","Ivanova",1,300);
+        thirdEmployee = new Employee("Sergey","Nikitich",2,500);
+        fourthEmployee = new Employee("Anna","Dmitrievna",2,700);
```

Comment:
имена переменных не помогают понять тесты ниже, у кого самая высокая ЗП, кто в каком отделе находится.
```suggestion
        firstDepMinSalaryEmployee = new Employee("Ivan","Ivanov",1,100);
        firstDepMaxSalaryEmployee = new Employee("Marina","Ivanova",1,300);
        secondDepMinSalaryEmployee = new Employee("Sergey","Nikitich",2,500);
        secondDepMaxSalaryEmployee = new Employee("Anna","Dmitrievna",2,700);
```